move/its/sources/interchain_transfer_ticket.move

```diff
@@ -0,0 +1,73 @@
+module its::interchain_transfer_ticket;
+
+use std::ascii::String;
+
+use sui::balance::Balance;
+
+use its::token_id::TokenId;
+
+/// -----
+/// Types
+/// -----
+#[allow(lint(coin_field))]
```

Comment:
what's the issue here?

Reply:
It goes away in a future PR, storing coins is unadvisable.